src/rgw/rgw_sal_filter.cc

```diff
@@ -1303,22 +1303,40 @@ int FilterWriter::complete(size_t accounted_size, const std::string& etag,
 			canceled, y);
 }
 
-int FilterLuaScriptManager::get(const DoutPrefixProvider* dpp, optional_yield y,
+int FilterLuaManager::get_script(const DoutPrefixProvider* dpp, optional_yield y,
 				const std::string& key, std::string& script)
 {
-  return next->get(dpp, y, key, script);
+  return next->get_script(dpp, y, key, script);
 }
 
-int FilterLuaScriptManager::put(const DoutPrefixProvider* dpp, optional_yield y,
+int FilterLuaManager::put_script(const DoutPrefixProvider* dpp, optional_yield y,
 				const std::string& key, const std::string& script)
 {
-  return next->put(dpp, y, key, script);
+  return next->put_script(dpp, y, key, script);
 }
 
-int FilterLuaScriptManager::del(const DoutPrefixProvider* dpp, optional_yield y,
+int FilterLuaManager::del_script(const DoutPrefixProvider* dpp, optional_yield y,
 				const std::string& key)
 {
-  return next->del(dpp, y, key);
+  return next->del_script(dpp, y, key);
+}
+
+int FilterLuaManager::add_package(const DoutPrefixProvider* dpp, optional_yield y,
+                                 const std::string& package_name)
+{
+  return -ENOENT;
```

Comment:
filters are supposed to forward all calls to the `next` store. this should `return next->add_package(...)`. same for remove_ and list_packages()

Reply:
done